### DISCUSSION TITLE
Kinetic Typography

### DISCUSSION THREAD

**Discussion by MrMikeMuzik** [keiwando/toonsquid#139] [Ideas]
I have recently purchased ToonSquid with the hope of using it to produce kinetic typography videos. Three features I would love to have added would be:
1)Ability to display measures and beats in the timeline by specifying tempo and time signature. 
2) Real-time marker button that would allow me to record or erase multiple markers in real time as the project plays instead of having to scrub the whole audio track to place dozens of markers one by one. 
Thank you for ToonSquid. It is obviously a labor of love. 
3) Import  Keynote .key files with ability to change the order and timing of builds.

**Comment by keiwando** [comment]
Thanks for the suggestions!
